tests/core/parallel.py

```
@@ -0,0 +1,30 @@
+    return os.environ.get("PYTEST_XDIST_WORKER", "master")
+
+
+def xdist_worker_index() -> int:
+    worker_id = xdist_worker_id()
+    if worker_id == "master":
+        return 0
+    return int(worker_id.removeprefix("gw"))
+
+
+def isolated_xdg_config_home() -> Path:
+    path = Path(os.environ.get("OSAC_TEST_TMP", "/tmp")) / "osac-e2e-config" / xdist_worker_id()
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)